client/ayon_core/hosts/houdini/plugins/create/create_model.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+"""Creator plugin for creating Model product type.
+
+Note:
+    Currently, This creator plugin is the same as 'create_pointcache.py'
+    But renaming the product type to 'model'.
+
+    It's purpose to support
+    Maya (load/publish model from maya to/from houdini).
+
+    It's considered to support multiple representations in the future.
+"""
+
+from ayon_core.hosts.houdini.api import plugin
+from ayon_core.lib import BoolDef
+
+import hou
+
+
+
+class CreateModel(plugin.HoudiniCreator):
+    """Create Model"""
+    identifier = "io.openpype.creators.houdini.model"
+    label = "Model"
+    product_type = "model"
+    icon = "cube"
+
+    def create(self, product_name, instance_data, pre_create_data):
+        instance_data.pop("active", None)
+        instance_data.update({"node_type": "alembic"})
+        creator_attributes = instance_data.setdefault(
+            "creator_attributes", dict())
+        creator_attributes["farm"] = pre_create_data["farm"]
+
+        instance = super(CreateModel, self).create(
+            product_name,
+            instance_data,
+            pre_create_data)
+
+        instance_node = hou.node(instance.get("instance_node"))
+        parms = {
+            "use_sop_path": True,
+            "build_from_path": True,
+            "path_attrib": "path",
+            "prim_to_detail_pattern": "cbId",
+            "format": 2,
+            "facesets": 0,
+            "filename": hou.text.expandString(
+                "$HIP/pyblish/{}.abc".format(product_name))
+        }
+
+        if self.selected_nodes:
+            selected_node = self.selected_nodes[0]
```

Comment:
is this intuitive?

Reply:
This is widely adopted in other creator plugins. I'm not justifying following the same idea in this creator. 
But, I just pointing out that it should be fixed in other creator plugins too! 
Therefore it's better discussed in a dedicated issue. 

Would you please create an issue about it so that it can be added to our backlog ?
I'd appreciate any suggested solutions or code snippets to furtherly improve it.

Reply:
I think most of these ROP nodes allow you passing multiple SOP nodes to export and not just a single one? And you'd expect at least a warning or something telling the user only one of the selected nodes is going to be used. However as I've said before I find the whole AYON/OP create workflow in Houdini very unintuitive so it's not just this small improvement IMO that needs fixing and there needs to be a much bigger redesign. I need to spend some good time trying to see how I can fit our workflow ideas in Houdini in Ayon but I don't really have much time atm as we already have a decently working solution for us

Reply:
> I find the whole AYON/OP create workflow in Houdini very unintuitive so it's not just this small improvement IMO that needs fixing and there needs to be a much bigger redesign. 

Yes, and I agree.. I'd appreciate any suggestions and I'll add them to my notes.